src/core.js

```diff
@@ -632,21 +632,8 @@ define([
                     label: "Form Name",
                     slug: "formName"
                 },
```

Comment:
(Careful. This trailing comma means that your array might be [one item longer][1] than you expect.)

  [1]: http://stackoverflow.com/a/7246662